File: Assignment 10/Activity 1.js
function main() {
    // this program uses a loop to generate a list of multiplication expressions for a given value
    var number;
    var factor;
    
    number = getValue("number");
    factor = getValue("factor");
    forLoop(number, factor);
}

function forLoop(number, factor) {
    window.alert(" While loop multiplication from " + number + " to " + factor + " by 1 :");
    var product;
    var increment;
    
    increment = 1;
    product = number;
    for (increment = increment; increment <= factor; increment++) {
        product = number * increment;
        window.alert(number.ToString() + " * " + increment + " = " + product);
    }
}

function getValue(name) {
    var value;
    
    window.alert(" Enter " + name + " value: ");
    value = window.prompt('Enter a value for value');
    
    return value;
}

File: Assignment 10/Activity 1.py
# this program that asks the user to enter grade scores and calculates the average of the entered scores.

def displayAverage(average):
    print(" The avergae of all these score is " + str(average))

    
def processScores():
    increment = 0
    sum = 0
    while True:    #This simulates a Do Loop
        score = getValue("score")
        if score >= 0:
            sum = sum + score
            print(sum)
            increment = increment + 1
        if not(score >= 0): break   #Exit loop
    average = float(sum) / increment
    return average

def getValue(name):
    print(" Enter " + name + " value: ")
    value = int(input())
    return value

def main():
    average = processScores()
    displayAverage(average)

    
main()
